app/models/miq_expression.rb

```diff
@@ -1101,7 +1100,7 @@ def self.value2human(val, options = {})
       friendly = tables.split(".").collect do|t|
         if t.downcase == "managed"
           val_is_a_tag = true
-          @company + " Tags"
+          "#{User.current_tenant.name} Tags"
```

Comment:
yes. it is the current user's tenant. I'm changing this to be the root tenant's name.